File: C/NestedLoops.c
#include <stdio.h>

int main(void) {
    int size;
    printf("Enter Size: ");
    scanf("%d", &size);
    for (int i = 0; i < size; i++) {
        for (int j = 0; j < size; j++) {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: Python/Nested Loops.py
# nested loops =  The "inner loop" will finish all of it's iterations before
#                 finishing one one iteration of the "outer loop"

rows = int(input("How many rows?: "))
columns = int(input("How many columns?: "))
symbol = input("Enter a symbol to use: ")

for i in range(rows):
    for j in range(columns):
        print(symbol, end="")          #prevents cursor from moving to next line
    print()
